### PR TITLE
Adds Icons Instead of Words for Some tabs (Enhancement)

### PR DESCRIPTION
**Pull request by CDAGaming**
## Description
To make the App more visually Appealing, some images have been added to the tab headers to replace words. This PR bc of the smaller tab sizes, adds support for more resolutions

### CODE REVIEW

**Comment by jjskuld** [comment]
@CDAGaming I think the idea of icons is cool, but i just don't like the icons that are in that screenshot. The red just doesn't match with the blue theme.  Also the icons seem a little pixellated.. maybe not high resolution icons?  In any case let's see if we can find better quality ones.

**Comment by CDAGaming** [comment]
@jjskuld Gotcha, I'll try to make higher resolution images tomorrow

**Comment by CDAGaming** [comment]
New Icons Will be Added today, Only Icon not changing it's style is the hub/pokeease(it'll just get resolution boost)

**Comment by CDAGaming** [comment]
Images made, will be added tomorrow

File: PoGo.Necrobot.Window/SettingsWindow.xaml.cs [hunk 1]
```diff
@@ -491,7 +490,7 @@ private void BtnSave_click(object sender, RoutedEventArgs e)
         private void MetroWindow_Closed(object sender, EventArgs e)
         {
             main.Visibility =  Visibility.Visible;
-           // this.Owner.Visibility = Visibility.Visible;
+            // this.Owner.Visibility = Visibility.Visible;
```
Comment (Lord-Haji) [inline PoGo.Necrobot.Window/SettingsWindow.xaml.cs]: You could just remove this

**Comment by jjskuld** [comment]
@CDAGaming Hmm.. can you also try making images greyscale? Just want to see how it looks when it's not red.

**Comment by hardknoxni69a** [comment]
is there something wrong with the words i believe these icons are not so pretty

**Comment by CDAGaming** [comment]
These Icons will be getting fully changed today. And Grey scale Previews will be sent too

**Comment by CDAGaming** [comment]
Screenshot Updated with Newer Images

**Comment by Lord-Haji** [comment]
PR Not Ready
Avoid Merging this @hardknoxni69a

**Comment by CDAGaming** [comment]
Build was Coming up with errors, Fixed!

**Comment by Lord-Haji** [comment]
@CDAGaming PR further stuff on the features/WINGUI_Ico

**Pull request merged**